src/styles/components/_Header.scss

```diff
@@ -86,3 +87,98 @@
     }
   }
 }
+
+/*
+----------------------------------------
+Homepage header
+----------------------------------------
+*/
+.hc-c-header--home .hc-c-menu {
+  background-color: $color-primary;
+}
+
+@media (min-width: $width-sm) {
+  .hc-c-header--home {
+    // On non-mobile screens, the homepage menu is always visible
+    .hc-c-menu {
+      display: block;
+    }
+
+    .hc-c-menu__content {
+      position: static;
+    }
+
+    .hc-c-menu__links li {
+      display: inline-block;
+    }
+  }
+}
+
+@media (max-width: $width-sm) {
+  // On small screens, the logo padding was creating a wonky focus style
```

Comment:
Updated spacing on small screens for a better focus style
<img width="426" alt="Screen Shot 2021-09-14 at 10 42 01 AM" src="https://user-images.githubusercontent.com/33579665/133299129-bb877f9e-2246-47f1-8e28-7b3e28cd2aed.png">

Reply:
Although it does fix the outline, it also ends up pulling the menu button to the left too far.

Reply:
🤦🏻‍♀️ Good catch.

Updated so Menu isn't pulled so far and looks like this
<img width="426" alt="Screen Shot 2021-09-14 at 12 19 58 PM" src="https://user-images.githubusercontent.com/33579665/133312376-e6a808f9-0f84-469d-94df-76011e63ddef.png">